toolkits/landcover/impl/landsat8.js

```diff
@@ -60,19 +64,27 @@ Landsat8.prototype = Object.create(Dataset.prototype);
 Landsat8.prototype.COMMON_BAND_NAMES = COMMON_BAND_NAMES;
 
 /**
- * Masks clouds and shadows using relevant bits in the `pixel_qa` band generated
- * by the CFMASK algorithm.
+ * Returns a copy of this dataset, copying all related state. Used internally
```

Comment:
If there's no extra state, is there a reason to override?

Reply:
If we enable type checking using our JSDoc we need to override this to create and return an instance of the right class.